extensions/wikia/GlobalWatchlist/GlobalWatchlistHooks.class.php

```diff
@@ -34,7 +34,6 @@ public static function savePreferences( &$formData, &$error ) {
 			$wg = \F::app()->wg;
 			$task = new GlobalWatchlistTask();
 			( new AsyncTaskList() )
-				->wikiId( $wg->CityId )
```

Comment:
We don't need the cityId for this since we're removing all entries for a user in the global_watchlist table, not ones based off a specfic wiki.